flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/table/KafkaConnectorOptions.java

```diff
@@ -248,6 +248,16 @@
                             "If the delivery guarantee is configured as "
                                     + DeliveryGuarantee.EXACTLY_ONCE
                                     + " this value is used a prefix for the identifier of all opened Kafka transactions.");
+    // --------------------------------------------------------------------------------------------
+    // Source specific options
+    // --------------------------------------------------------------------------------------------
+
+    public static final ConfigOption<Long> SOURCE_BOUNDED =
+            ConfigOptions.key("source.Bounded")
+                    .longType()
+                    .noDefaultValue()
```

Comment:
The default value of the boundness is `CONTINUOUS_UNBOUNDED` in `KafkaSourceBuilder`. Therefore does this have the default value?